src/components/InvertedFlatList/index.ios.js

```diff
@@ -1,11 +1,20 @@
 import React, {forwardRef} from 'react';
+import PropTypes from 'prop-types';
 import BaseInvertedFlatList from './BaseInvertedFlatList';
 
-export default forwardRef((props, ref) => (
+const propTypes = {
+    /** The initial number of items to render */
+    initialNumToRender: PropTypes.number.isRequired,
+};
+
+const InvertedFlatList = forwardRef((props, ref) => (
     <BaseInvertedFlatList
         // eslint-disable-next-line react/jsx-props-no-spreading
         {...props}
         ref={ref}
-        initialNumToRender={20}
+        initialNumToRender={props.initialNumToRender}
```

Comment:
Probably we can just remove this since each platform version is getting the prop passed via `...props` same for the android, web, versions. 

I'm not sure it should be a required prop as we can still fallback to the default value in cases where we are not getting more specific. Other things may use the inverted `FlatList` without this optimization.

Reply:
🤦‍♂️ Thanks for the catch! This really is unnecessary, changed the code for all platforms.